packages/ragbits-cli/src/ragbits/cli/__init__.py

```diff
@@ -2,11 +2,25 @@
 import pkgutil
 from pathlib import Path
 
-from typer import Typer
+import typer
 
 import ragbits
 
-app = Typer(no_args_is_help=True)
+from .app import CLI, OutputType
+
+app = CLI(no_args_is_help=True)
+
+
+@app.callback()
+def output_type(
+    output: OutputType = typer.Option("text", "--output", "-o", help="Set the output type (text or json)"),  # noqa: B008
+) -> None:
+    """Sets an output type for the CLI
+    Args:
+        output: type of output to be set
+    """
+    app.set_output_type(output_type=output)
+    print(f"Output type set to {output}")
```

Comment:
I guess this is not intentional? Especially when outputting json it's important to be otherwise quiet, so that the json output can be parsed by other commands

Reply:
this was for debuging purpose - removed